COMMERCE-12929 - Guest provided address saved and cached in and authenticated user is offered to use it

victorcdp opened the pull request:
Related Issue: [COMMERCE-12929](https://liferay.atlassian.net/browse/COMMERCE-12929)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

noornajj commented:
ci:test:sf

noornajj commented:
ci:test:relevent

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-10.liferay.com/job/test-portal-source-format/2514/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-10.liferay.com//userContent/jobs/test-portal-source-format/builds/2514/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/4321/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-10.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4321/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-127800">LPS-127800</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/03ea780079850c51dfe8ad07142cdc8a257eea01">03ea780079850c51dfe8ad07142cdc8a257eea01</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-source-format/2514/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/2514/jenkins-report.html">here</a>.</h5><pre><code>     [java] java.lang.Exception: Found 2 formatting issues:
     [java] 1: For better performance, the variable declaration for 'referringPortletResource' should be moved inside the if-statement on line '407': ./modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java 404 (Checkstyle:VariableDeclarationAsUsedCheck)
     [java] 2: ./modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/dynamic_include/init.jsp expected:&lt;....liferay.journal.con[]tent.web.internal.co...&gt; but was:&lt;....liferay.journal.con[stants.JournalContentPortletKeys" %&gt;&lt;%@
     [java] page import="com.liferay.journal.con]tent.web.internal.co...&gt;
     [java] 
     [java] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:445)
     [java] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:290)
[stopwatch] [run.batch.test.action: 1:24.376 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:572: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:706: Java returned: 1
      [get] Getting: http://test-1-10/job/test-portal-source-format/2514//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20210217134757436.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20210217134757436.txt
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null741380294.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1843396238
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 211ms
[junitreport] Deleting: /tmp/null1843396238
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results</code></pre></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevent - 0 out of 1 jobs passed in 0 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1558">Acceptance Upstream DXP (master) #1558</a></p><details><summary><strong>ci:test:relevent - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/4321/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4321/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/4321/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The CI test suite 'relevent' is not a valid test suite.

List of available CI test suites:
* acceptance
* acceptance-upstream
* accounts
* analytics-cloud
* app-builder
* auto-tagging
* backend
* backend-hypersonic
* blade
* blogs
* bulk-editing
* bundle
* calendar
* clustering
* commerce
* content-page-review
* data-engine
* depot
* dl-store
* documents
* drools
* dummy
* echo
* echo-acceptance
* echo-upgrade
* environments
* environments-alpine38
* environments-alpine310
* environments-centos6
* environments-centos8
* environments-debian7
* environments-debian8
* environments-debian9
* environments-oracle19
* environments-orcllinux6
* environments-orcllinux7
* environments-postgresql11
* environments-release-ce
* environments-release-dxp
* environments-suse15
* environments-ubuntu14
* environments-ubuntu16
* environments-ubuntu18
* export-import
* forms
* frontend
* gauntlet
* gradle-plugins
* headless
* headless-acceptance
* jdk11
* jsf
* knowledge-base
* ldap
* license
* lima
* lpkg
* message-boards
* oauth2
* onedrive
* online-editing
* poshi
* poshi-gauntlet
* project-templates
* publications
* redirect
* relevant
* reports
* roles
* saml
* search
* search-jdk11
* security
* seo
* sf
* sharepoint
* sharing
* site-template
* stable
* staging
* sybase
* system
* tango
* tango-acceptance
* tango-analytics-cloud
* tck
* translations
* upgrade
* upgrade-client
* users
* wiki
* workflow

</p><p>Comment 'ci:help' for details.</p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-10.liferay.com/job/test-portal-source-format/2514/">test-portal-source-format#2514</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/2514/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/124">brianikim#124</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/92155.aspx">master - noornajj &gt; brianikim - PR#124 - 2021-02-17[13:45:23]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/16001/">publish-spira-report#16001</a><br />

noornajj commented:
ci:test:sf

noornajj commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-4.liferay.com/job/test-portal-source-format/4246/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-source-format/builds/4246/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/7550/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7550/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-127800">LPS-127800</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/03ea780079850c51dfe8ad07142cdc8a257eea01">03ea780079850c51dfe8ad07142cdc8a257eea01</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-source-format/4246/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-source-format/builds/4246/jenkins-report.html">here</a>.</h5><pre><code>	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:352)
	at net.sf.antcontrib.logic.TryCatchTask.execute(TryCatchTask.java:207)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:352)
	at org.apache.tools.ant.Target.execute(Target.java:437)
	at org.apache.tools.ant.Target.performTasks(Target.java:458)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1406)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1377)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1261)
	at org.apache.tools.ant.Main.runBuild(Main.java:857)
	at org.apache.tools.ant.Main.startAnt(Main.java:236)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
[beanshell] Created cache file in /tmp/jenkins-cached-files/1725132267.txt

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:12587: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-portal-source-format.xml:18: Sourced file: inline evaluation of: `` 			 				import com.liferay.jenkins.results.parser.AntUtil; 				import com.lifer . . . '' : TargetError : at Line: 324 : in file: inline evaluation of: `` 			 				import com.liferay.jenkins.results.parser.AntUtil; 				import com.lifer . . . '' : throw e ; 

Target exception: java.lang.RuntimeException: Unable to create local branch noornajj-124-LPS-127800 at 03ea780079850c51dfe8ad07142cdc8a257eea01
fatal: Not a valid branch point: '03ea780079850c51dfe8ad07142cdc8a257eea01'.</code></pre></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-4.liferay.com/job/test-portal-source-format/4246/">test-portal-source-format#4246</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-source-format/builds/4246/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/124">brianikim#124</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/92198.aspx">master - noornajj &gt; brianikim - PR#124 - 2021-02-17[15:19:06]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/16007/">publish-spira-report#16007</a><br />

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 23 out of 23 jobs passed in 1 hour 32 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a9876d03bc1aa264ebc142553b0f7db9c6901cbe">a9876d03bc1aa264ebc142553b0f7db9c6901cbe</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1558">Acceptance Upstream DXP (master) #1558</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459434/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459438/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1327/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/488263/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459435/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/377294/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459436/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425629/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453394/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 23 out of 23 jobs PASSED</strong></summary><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/7550/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463920/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425631/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459434/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414802/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425633/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459438/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1327/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/488264/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/488262/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453393/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453395/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/488263/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425630/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425632/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459435/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414801/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/377294/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459436/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463921/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/425629/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414800/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453394/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7550/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/7550/">test-portal-acceptance-pullrequest(master)#7550</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7550/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/124">brianikim#124</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2004.aspx">/Liferay DXP 7.4/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/92211.aspx">master - noornajj &gt; brianikim - PR#124 - 2021-02-17[15:19:12]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-4.liferay.com/job/publish-spira-report/15884/">publish-spira-report#15884</a><br />

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/4321/">test-portal-acceptance-pullrequest(master)#4321</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4321/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevent<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/124">brianikim#124</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2459.aspx">/Liferay DXP 7.4/Pull Request/ci:test:relevent</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/92233.aspx">master - noornajj &gt; brianikim - PR#124 - 2021-02-17[13:45:38]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-8.liferay.com/job/publish-spira-report/14062/">publish-spira-report#14062</a><br />

thaynam committed:
ISSITES-6112 Create Learn site initializer

qz1010 committed:
LPS-196539 SF rename variable name

pedro-oliveira446 committed:
LPS-195680 Add new method with system parameter

pedro-oliveira446 committed:
LPS-195680 Test add system object validation rule

pedro-oliveira446 committed:
LPS-195680 Test delete system object validation rule

pedro-oliveira446 committed:
LPS-195680 Test update system object validation rule

pedro-oliveira446 committed:
LPS-195680 Add new method to create a modifiable system object definition

BarbaraCabrera committed:
LPS-200208 Change the getMostRelevantLocale() to the getDefault() to use the site language instead of the user one for Fragments

BarbaraCabrera committed:
LPS-199573 Change the getMostRelevantLocale() to the getDefault() to use the site language instead of the user one for Journal

BarbaraCabrera committed:
LPS-200209 Change the getMostRelevantLocale() to the getDefault() to use the site language instead of the user one for Stylebook

BarbaraCabrera committed:
LPS-200220 Change the getMostRelevantLocale() to the getDefault() to use the site language instead of the user one for Utility Pages

BarbaraCabrera committed:
LPS-200214 Change the getMostRelevantLocale() to the getDefault() to use the site language instead of the user one for Page Templates

MirnaGama committed:
LPS-200153 Add user with workflow for edit user action command class

DiegoHu97 committed:
LPS-199736 Create react component for Invite Members widget

DiegoHu97 committed:
LPS-199736 Imports

DiegoHu97 committed:
LPS-199736 Remove not needed code

MicaelleSilvaa committed:
LPS-200168 Update replacements.json

victorg1991 committed:
LPS-200776 Save conditions, actions and condition type

victorg1991 committed:
LPS-200776 Add condition type, conditions and action to the schema

MicaelleSilvaa committed:
LPS-200168 Add new test cases for UpgradeCatchAllCheck

MicaelleSilvaa committed:
LPS-200168 Delete UpgradeJavaFetchAssetCategoryByExternalReferenceCodeCheck

victorg1991 committed:
LPS-200776 buildRest

victorg1991 committed:
LPS-200776 Add condition and actions to import/export

Commit:
LPS-196122 Replace ServiceProxyFactory with dynamic Snapshot in class MessageBusUtil

danielCamara02 committed:
LPS-199989 Improve and rename CanPublishLocalizedContent test

danielCamara02 committed:
LPS-199989 Remove CanPublishLocalizedContent from Publications testcase

MicaelleSilvaa committed:
LPS-200168 Generated

DiegoHu97 committed:
LPS-199736 buildLang

Commit:
LPS-201399 Missing implementation of UserManager#listUsersWithGET.

For LPS-97345.

Mikellorza committed:
LPS-201547 Refactor code

rafaprax committed:
LPS-201399 Add schemas property to all SCIM Resources according to RFC, https://datatracker.ietf.org/doc/html/rfc7643#section-3 .

For LPS-97345.

rafaprax committed:
LPS-201399 Build rest.

For LPS-97345.

Commit:
ISSD-1499 Fixed Dashboard - the 'Expiring Soon' field is not showing the value

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in lib

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in build-test.xml

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in modules

jeffwu0724 committed:
LPS-201428 upgrade setup.database.jar.url and setup.database.jar.sha1 in portal.properties

Mikellorza committed:
LPS-201547 Modify count assets tag usages with case-sensitive

Commit:
ISSD-1499 Undo changes in the MDFRequestChart

Commit:
ISSD-1499 Expiring Soon field is not showing the value

rafaprax committed:
LPS-201399 Move endpoint configuration to a static block. The AbstractResourceManager is used internally by the charon lib into the parser request process.

For LPS-97345.

rafaprax committed:
LPS-201399 Fix typo.

For LPS-97345.

rafaprax committed:
LPS-201399 Set id property in the ScimUser model.

For LPS-97345.

victorg1991 committed:
LPS-200776 Add missing id creation

victorg1991 committed:
LPS-200776 Add id to the page element definition

Mikellorza committed:
LPS-201547 Improve code

victorg1991 committed:
LPS-200776 buildRest

victorg1991 committed:
LPS-200776 Allow creating layout structure items with a given id

victorg1991 committed:
LPS-200776 Take id into consideration in the import/export

qz1010 committed:
LPS-196539 SF

Mikellorza committed:
LPS-201547 Move method from jsp to display context

BarbaraCabrera committed:
LPS-199704 Adapt Integration tests

victorg1991 committed:
LPS-200776 Include available segments entries in all type of pages since it is needed for the rules

victorg1991 committed:
LPS-200776 Do not include dropzones

jkappler committed:
LPS-200442 New comparator for localized title of a category taking into account the vocabulary. We take AssetVocabularyGroupLocalizedTitleComparator as an example

jkappler committed:
LPS-200442 Sort vocabularies by locale.

jkappler committed:
LPS-200442 Sort categories by locale.

carolmariaabb committed:
LPS-201695 Add new AutoIncrementObjectFieldBuilder

carolmariaabb committed:
LPS-201695 baseline

pedro-oliveira446 committed:
LPS-195680 Test put object definition with custom and system object validation rule

jeffwu0724 committed:
LPS-201428 auto generate, by running "ant build-lib-versions"

pedro-oliveira446 committed:
LPS-195680 Refactored to null assignment in preparation for clearing feature flag properties

Andrefes565 committed:
ISSD-1364 Move to liferay-learn-workspace

pedro-oliveira446 committed:
LPS-195680 Create method to assert object validation rule changes

pedro-oliveira446 committed:
LPS-195680 Use new method

Commit:
ISSD-1679 Tooltip creation and download keys error message

rafaprax committed:
LPS-201399 Throw Charon specific exceptions instead of portal ones and let the charon handle the exception properly.

For LPS-97345.

carolmariaabb committed:
LPS-201695 Add integration tests for AutoIncrement field

Commit:
LPS-196764 Define the right color for draft status

rafaprax committed:
LPS-201399 Transform _fetchScimUser in a getter method and apply specific validation logic to it.

For LPS-97345.

rafaprax committed:
LPS-201399 Decorate the charon UserResourceManagerI implementation to give support to handle the ConflictExpection into the updateWithPUT method. it was required to cover the the AC-15 from the https://liferay.atlassian.net/browse/LPS-97345 .

For LPS-97345.

kenjiheigel committed:
LRCI-3881 Init module as gradle project

kenjiheigel committed:
LRCI-3881 Generate playwright files

kenjiheigel committed:
LRCI-3881 Add sample test

pei-jung committed:
LPS-202050 Use browser default time input

Mikellorza committed:
LPS-201547 Improve code

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagServiceImpl.java -->  getVisibleAssetsTagsCount(long groupId, String name)

Replace by method getVisibleAssetsTagsCount(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java --> getTagsSize(long groupId, String name)

Replace by method getTagsSize(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/persistence/impl/AssetTagFinderImpl.java --> countByG_N(long groupId, String name)

Replace by method countByG_C_N(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 buildService

Mikellorza committed:
LPS-201547 Adapt usages of deleted methods

Commit:
LPS-201399 Expando table created for wrong model class (fixes LPS-201399).

For LPS-97345.

Mikellorza committed:
LPS-201547 baseline

victorg1991 committed:
LPS-201532 Generate an id in case it is null

victorg1991 committed:
LPS-200776 Set the id of the root element when importing

victorg1991 committed:
LPS-200776 When creating the collection, a collection item is created so allow passing the id of it

victorg1991 committed:
LPS-200776 Adapt tests

kenjiheigel committed:
LRCI-3881 Add playwright docker macrodefs

kenjiheigel committed:
LRCI-3881 Add playwright batch

kenjiheigel committed:
LRCI-3881 Add playwright test suite

susanchen3 committed:
LPS-201852 Temp ignore until LPS-195673 is fixed

Andrefes565 committed:
ISSD-1364 sf

Luizcarlosqueiroz committed:
LPS-201236 Navigate to another tab

AliciaGarciaGarcia committed:
LPS-201887 check if sharing is enabled

victorg1991 committed:
LPS-200776 Id is not readOnly

victorg1991 committed:
LPS-200776 buildRest

victorg1991 committed:
LPS-200776 Add tests for rules

BarbaraCabrera committed:
LPS-200208 Add Integration test for Copying fragments

Commit:
LPS-197484 fix testAccountEntryObjectValidationRule in AccountEntryLocalServiceTest

Commit:
LPS-197484 fix testOrganizationObjectValidationRule in OrganizationLocalServiceTest

michaelhashimoto committed:
LRCI-3823 Fix errors found on startup

interaminense committed:
LRAC-15035 Display no results found when there are data but no filtered items

michaelhashimoto committed:
LRCI-3823 Add a GitHubClient object that send messages to GitHub

michaelhashimoto committed:
LRCI-3823 Add an event handler for ci:help

michaelhashimoto committed:
LRCI-3823 Add an event handler for ci:test

qz1010 committed:
LPS-201984 Enforce variable name for 'JsonNode' and 'ObjectNode'

qz1010 committed:
LPS-201984 SF

JoyceWang08211 committed:
LPS-199771 Add macro to configure SCIM via UI

JoyceWang08211 committed:
LPS-199771 Add test

jaydawu committed:
LPS-200153 Add test CanReceiveNewUserNotificationsWhenSingleApproverWorkflowIsAppliedToUser

ling-alan-huang committed:
LPS-201984 Enforce method name

ling-alan-huang committed:
LPS-199234 Fix pattern

brianikim committed:
COMMERCE-12864 Do not merge guest orders in pending orders view

brianikim committed:
COMMERCE-12864 Remove session attribute when order is guest order

Hijky committed:
LPS-182406 Add a macro to edit Job Scheduler item

Commit:
LPS-201399 Expando table read for wrong model class also.

For LPS-97345.

victorg1991 committed:
LPS-200776 Don't assign it if it is null, we need a nonnull id

AliciaGarciaGarcia committed:
LPS-200620 change message depends if number of failures > 1

DiegoHu97 committed:
LPS-202014 Add title to back button when importing display page templates

AliciaGarciaGarcia committed:
LPS-200620 buildLang

victorg1991 committed:
LPS-200776 Use util

ambrinchaudhary committed:
LPS-199736 visual improvements

ambrinchaudhary committed:
LPS-199736 auto SF - identation

ambrinchaudhary committed:
LPS-199736 Move to an input group

ambrinchaudhary committed:
LPS-199736 Use the ids array

ambrinchaudhary committed:
LPS-199736 remove unnecessary state

davisaints committed:
LPS-201324 Locate the inline option

davisaints committed:
LPS-201324 Remove unnecessary macros

davisaints committed:
LPS-201324 Make assertions at the test case level

ling-alan-huang committed:
LPS-196539 Semantic versioning

AliciaGarciaGarcia committed:
LPS-200900 add missing references to help message.
 Links seems not been working on help message, so I omitted it

ambrinchaudhary committed:
LPS-199736 Fix typo

ambrinchaudhary committed:
LPS-199736 React:component already provide a portletNamespace

kenjiheigel committed:
LRCI-3881 Format & update playwright config

samuelleolopes committed:
ISSD-1718 Added permission for the administrator user to generate license without being associated with the project

michaelhashimoto committed:
LRCI-3823 Pull the Job Queue from the cache instead of the database

joseabelenda committed:
ISSD-1364 No longer necessary

joseabelenda committed:
ISSD-1364 Sort

Luizcarlosqueiroz committed:
LPS-188421 Fix and impove test assertion

interaminense committed:
LRAC-15047 Create unit tests for Download CSV Report component

interaminense committed:
LRAC-15038 Create unit tests for Download CSV Individuals Report component

interaminense committed:
LRAC-15049 Create unit tests for Download PDF Individuals Report component

fmarcio committed:
LRAC-15072 to LRAC-15077 update lang key

fmarcio committed:
LRAC-15072 to LRAC-15077 apply update langs on DownloadCSVReport component

allen-nagtalon committed:
LPS-200084 Create test to assert draft of workflow enabled entity cannot be published

kenjiheigel committed:
LRCI-3881 Rename package

kenjiheigel committed:
LRCI-3881 Auto SF

michaelhashimoto committed:
LRCI-3890 Change the jethr0 client interactions for new build parameters format

joseabelenda committed:
ISSD-1364 SF

michaelhashimoto committed:
LRCI-3890 Fix the parameters passed when invoking a build

victorg1991 committed:
LPS-200776 buildRest

Hijky committed:
LPS-182406 Add a macro to create Job Scheduler by json

Hijky committed:
LPS-182406 Add a testcase to switch to Talend tab in Job Scheduler wihout errors in console

JoyceWang08211 committed:
Revert "LRQA-84306 Update macro to support multiple tomcat instances"

This reverts commit d80cac23af80d5191b013cf7488989e440ee60a1.

JoyceWang08211 committed:
Revert "LRQA-84306 Replocate osgi config to global dependencies folder as it's used by multiple component tests"

This reverts commit 510c27cf6f25b714ec7ea9dfacf4ef92736fb7d4.

JoyceWang08211 committed:
Revert "LRQA-84306 Extend build xml to support additional tomcat instances"

This reverts commit e0b1ba8760cb1b29883615e3ea52a572f302973b.

Commit:
LPS-194606 Remove tear down block

Commit:
LPS-201911 Update step to review changes

Commit:
LPS-194606 Add function to remove element attribute

Commit:
LPS-194606 Add case to view file upload field error message when partial validation failure

Commit:
LPS-194606 Remove tear down block

Commit:
LPS-194606 Add case to view multiselect list field error message when partial validation failure

Commit:
LPS-194606 Remove tear down block

Commit:
LPS-194606 Add case to view numeric input field error message when partial validation failure

Commit:
LPS-194606 Update description

haydenzheng314 committed:
LPS-202093 Replace non-blocking ServiceProxyFactory usages with Snapshot in class AggregateFilter

Commit:
LPS-194606 Remove tear down block

Commit:
LPS-194606 Use headless api to instead of UI steps

Commit:
LPS-194606 Add locator for error on form

Commit:
LPS-194606 Modify case to view form error

Commit:
LPS-194606 Add case to view rich text input field error message when partial validation failure

Commit:
LPS-194606 Add case to view select from list field error message when partial validation failure

JanisZhang committed:
LPS-202096 Replace blocking ServiceProxyFactory with Snapshot in class AutoBatchPreparedStatementUtil

robertoDiaz committed:
LPS-201923 Remove ".index" files when necessary

ccorreagg committed:
LPS-202102 Throw an IllegalArgumentException if the base64 encoding is wrong

JessieZhao0714 committed:
LRQA-83161 Add a pause to wait for scheduled publications to be published

Youness-ELMO committed:
LPS-190596 - Automation to Allow users to define sorts

victorg1991 committed:
LPS-201948 Pass a copy to avoid mutating the original array

AliciaGarciaGarcia committed:
LPS-202139 get the asset Rendered on the jsp to reuse and check

andreluis2022 committed:
LPS-200216 Adding new paths to actions

andreluis2022 committed:
LPS-200216 Updating the macro addActions

andreluis2022 committed:
LPS-200216 Creating a new file DataSetViewCreationActions

andreluis2022 committed:
LPS-200216 SF

Commit:
LPS-201399 SemVer

ambrinchaudhary committed:
LPS-201919 Make the horizontal card a focusable element

Commit:
LPS-200919 buildREST

Commit:
LPS-192109 Remove tear down block

Commit:
LPS-192109 Modify case to make it stable

brianchandotcom committed:
LPS-196122 SF

brianchandotcom committed:
LPS-201399 Sort

Commit:
LPS-192664 Modify case to make it stable

brianchandotcom committed:
LPS-201399 Sort

brianchandotcom committed:
LPS-201399 add @Override

Commit:
LPS-193441 Remove tear down block

brianchandotcom committed:
LPS-201399 I think SZ did this so it doesn't happen during startup but only when it is used

brianchandotcom committed:
LPS-201399 Like this

brianchandotcom committed:
LPS-201399 Correct author

brianchandotcom committed:
LPS-201399 It's no longer fetch but get

Commit:
LPS-193441 Modify case to make it stable

brianchandotcom committed:
LPS-200900 I believe 'true' will do the trick.

brianchandotcom committed:
LPS-201695 SF

brianchandotcom committed:
LPS-199573 SF

brianchandotcom committed:
LPS-199573 Sort

brianchandotcom committed:
LPS-199573 SF

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-201923 SF

AliciaGarciaGarcia committed:
LPS-202139 autoSF

AliciaGarciaGarcia committed:
LPS-202139 show status if now visible and do not show the link on that case

interaminense committed:
LRAC-15075 buildLang

Commit:
LPS-202183 Add feature flag to cases related to locked pages

pyoo47 committed:
LRCI-3888 Reorganize ParallelExecutor to allow users to process in parallel, multiple lists of callables that are run sequentially.

brianchandotcom committed:
LPS-202102 Wordsmith

brianchandotcom committed:
LPS-201399 PMD error for unassigned var scimUser

brianikim committed:
COMMERCE-12864 Attribute needs to be in *WebKeys

kenjiheigel committed:
LRCI-3881 Stabilize connection by using Jenkins node hostname with the playwright container

xbrianlee committed:
LPS-198602 Order matters

xbrianlee committed:
LPS-198602 Add suppression

brianikim committed:
COMMERCE-12864 Autogenerated (baseline)

kenjiheigel committed:
LRCI-3881 Skip playwright dir for ModulesStructureTest

pyoo47 committed:
LRCI-3888 Fix broken ParallelExecutor usages

pyoo47 committed:
LRCI-3888 Set http connection and read timeouts to 5 minutes

pyoo47 committed:
LRCI-3888 The first step of run<Status>() should not be to set the status to a status that it already has.

pyoo47 committed:
LRCI-3888 Remove debug code

pyoo47 committed:
LRCI-3888 Don't create a parallel task for a build that is complete

brianchandotcom committed:
LRCI-3890 prep next

pyoo47 committed:
LRCI-3888 Don't call getBuildMessage() unless the status has changed

brianchandotcom committed:
LPS-195680 SF

kenjiheigel committed:
LRCI-3835 Break out of the loop properly

xbrianlee committed:
LPS-202198 Type 12 hour UTC time

Commit:
LPS-202177 Remove duplicates macro

rafaprax committed:
Revert "LPS-201399 I think SZ did this so it doesn't happen during startup but only when it is used"

This reverts commit aebcd2d355f37fe52f557653e1e8411ea9d5c205.

Commit:
LPS-202177 Remove property that skip testcase on virtual instance

Commit:
LPS-202177 Add saves to avoid errors on CI

brianchandotcom committed:
LRCI-3835 prep next

brianchandotcom committed:
Revert "LPS-200065 Fix for LPKG builds"

This reverts commit 7209d25be51ef2ad8cf90fc1655c01cfc7f2312f.

brianchandotcom committed:
Revert "LPS-200065 Readme"

This reverts commit 24166856bd639321b3804cfb7846b03cf38b384a.

brianchandotcom committed:
Revert "LPS-200065 Set default deploy directory to osgi/portal"

This reverts commit 3808e5d5dba5a7554bb1ef14a608c2c3d4149f43.

brianchandotcom committed:
LPS-200065 prep next

brianchandotcom committed:
LPS-201547 Rename

Commit:
COMMERCE-12929 reset order addresses if changing from guest to user

pyoo47 committed:
LRCI-3888 Don't look for the build in the queue immediately

pyoo47 committed:
LRCI-3888 Check running task count + remaining task count

pyoo47 committed:
LRCI-3888 Increase timeout to 15 minutes

pyoo47 committed:
LRCI-3888 Count actual running callables

brianchandotcom committed:
LRCI-3888 Wordsmith

brianchandotcom committed:
LRCI-3888 prep next

Commit:
LPS-199849 Add case to filter attachment from computer by file type

Commit:
LPS-199849 Add case to filter attachment from documents and media by file type

Commit:
LPS-199849 Add case to assert cannot drag and drop invalid file type to documents and media modal

brianchandotcom committed:
LPS-202096 as used

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8bde4e6b402e6873fa80b7a69e0fca6c00bfe570">8bde4e6b402e6873fa80b7a69e0fca6c00bfe570</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-12929">COMMERCE-12929</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/ac3adf2f895cedac7f661045dc1b8eb20e7dc855">ac3adf2f895cedac7f661045dc1b8eb20e7dc855</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-source-format/2873/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/2873/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/2873/">test-portal-source-format#2873</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/2873/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/124">brianikim#124</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2462935052">[master] ci:test:sf - victorcdp &gt; brianikim - PR#124 - 2023-11-21[08:34:41]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/21642/">publish-testray-report#21642</a><br/></div></div>

brianikim commented on modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/CommerceOrderHttpHelperImpl.java:
```diff
@@ -582,9 +582,15 @@ private CommerceOrder _checkGuestOrder(
+				commerceOrder.getCommerceOrderId(), 0);
+			commerceOrder = _commerceOrderLocalService.updateBillingAddress(
+				commerceOrder.getCommerceOrderId(), 0);
+
```
You are re-indexing twice unnecessarily if you do this. Following is better if we do not need to reindex:

```
commerceOrder.setBillingAddressId(0);
commerceOrder.setShippingAddressId(0);

 _commerceOrderLocalService.updateAccount(
				commerceOrder.getCommerceOrderId(), user.getUserId(),
				accountEntry.getAccountEntryId());

```

victorcdp replied:
set*Address methods only update the object, the db entity is not updated. So when updateAccount is called, the order is fetched with the original addresses' ids. We need to find a way to update the address id values in the db as well

victorcdp replied:
@brianikim Could we make a resetAddresses method like [resetTermsAndConditions](https://github.com/liferay/liferay-portal/blob/6c18f8775a043f2ad6f56daddda998a2542acfef/modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java#L1056)?
It would set the addresses' ids to 0 and update in the db
